### PR TITLE
Fixed #30808 -- Added the Django Forum to contributing index.

### PR DESCRIPTION
**Pull request by grahamhealy20**
Adding a new bullet to the contributing page about the [Django forum](https://forum.djangoproject.com/) as per [#30808](https://code.djangoproject.com/ticket/30808#comment:6)

### CODE REVIEW

**Comment by carltongibson** [comment]
This is pending final approved from the DSF board, but thanks (in the meantime).

**Comment by andrewgodwin** [comment]
The DSF board has approved the funding, so please go ahead and land this!

File: docs/internals/contributing/index.txt [hunk 1]
```diff
@@ -24,6 +24,10 @@ contribute in many ways:
   ecosystem of pluggable applications is a big strength of Django, help us
   build it!
 
+* Join the `Django forum`_. This is a place for discussing the Django framework
+  and applications and projects that use it. This will be a good place to ask
+  questions or join in the discussion about contributing to the Django project.
```
Comment (felixxm) [inline docs/internals/contributing/index.txt]: I would move this point below the _"#django IRC channel"_ and make a small rewording:
```
* Join the `Django forum`_. This forum is a place for discussing the Django
  framework and applications and projects that use it. This is also a good
  place to ask and answer any questions related to installing, using, or
  contributing to Django.
```

Comment (carltongibson) [reply docs/internals/contributing/index.txt]: What about top of the list, above mailing list and IRC? (@felixxm You say!)

👍 on @felixxm's suggested wording.

Comment (felixxm) [reply docs/internals/contributing/index.txt]: Agreed, it can go to the top of the list :1st_place_medal:

Comment (grahamhealy20) [reply docs/internals/contributing/index.txt]: Perfect, will make the required adjustments and update this PR. Thanks all!

**Comment by felixxm** [comment]
@grahamhealy20 Thanks :+1: I updated wording.

Welcome aboard :boat:

**Pull request merged**